spring-cloud-dataflow-server/docker-compose-dood.yml

```diff
@@ -33,7 +33,7 @@ services:
       - SPRING_CLOUD_DATAFLOW_TASK_PLATFORM_LOCAL_ACCOUNTS_DEFAULT_DOCKER_NETWORK=${COMPOSE_PROJECT_NAME:?COMPOSE_PROJECT_NAME must be set when DooD is enabled}_default
       - SPRING_CLOUD_DATAFLOW_TASK_PLATFORM_LOCAL_ACCOUNTS_DEFAULT_DOCKER_DELETE_CONTAINER_ON_EXIT=${DOCKER_DELETE_CONTAINER_ON_EXIT:-true}
       # Override the CTR form maven to docker
-      - SPRING_CLOUD_DATAFLOW_TASK_COMPOSED_TASK_RUNNER_URI=docker://springcloud/spring-cloud-dataflow-composed-task-runner:${DATAFLOW_VERSION}
+      - SPRING_CLOUD_DATAFLOW_TASK_COMPOSED_TASK_RUNNER_URI=docker://springcloud/spring-cloud-dataflow-composed-task-runner:${DATAFLOW_VERSION:-latest}
```

Comment:
The `latest` is not a valid image tag nether for the SCDF or Skipper server nor for the Stream or Task apps.  An explicit app version tags is required.

Reply:
So this is not always latest, you can supply the version either as a variable `${DATAFLOW_VERSION}` or if nothing is specified then the `latest` is chosen for local quick testing.